src/Post/Post.js

```diff
@@ -359,9 +364,88 @@ function CommentSection({
     </section>
   );
 }
+
+function MediaSection({ media_items, isExpanded }) {
+  const mediaContainer = React.useRef();
+  const [mediaSectionWidth, setMediaSectionWidth] = React.useState(0);
+  const [mediaIndex, setMediaIndex] = React.useState(0);
+  const scrollLeft = -1 * mediaIndex * mediaSectionWidth;
+
+  return (
+    <section
+      className="media-section"
+      ref={(ref) => setMediaSectionWidth(elementWidth(ref))}
+    >
+      <div
+        className="media-container"
+        ref={mediaContainer}
+        style={{ transform: `translateX(${scrollLeft}px)` }}
+      >
+        {media_items?.map((mediaItem, index) => (
+          <div key={index}>
+            {mediaItem.type === 'photo' ? (
+              <PostImage imageURL={mediaItem.url} />
+            ) : (
+              <PostVideo
+                videoURL={mediaItem.url}
+                active={mediaIndex === index}
+              />
+            )}
+          </div>
+        ))}
+      </div>
+      {media_items.length > 1 && (
+        <div className="img-buttons-container">
+          <button
+            className={cx('prev-img-button', { hidden: scrollLeft === 0 })}
+            onClick={() => {
+              setMediaIndex((mediaIndex) => {
+                if (mediaIndex > 0) {
+                  return mediaIndex - 1;
+                }
+              });
+            }}
```

Comment:
Since the button will be hidden, you don't need to handle the edge cases:
```suggestion
            onClick={() => setMediaIndex(mediaIndex - 1)}
```